### ISSUE TITLE
[Accessibility][Usable-Homepage]: Whole navigation is provided with tab key within the left navigation pane.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2040]
**User Experience**
For better usability purpose, whole navigation should not be provided with tab key within the left navigation pane as it will increase the efforts of users to completely navigate with tab key & then to reach to main section.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. Observe the issue within the left navigation pane.

**Actual:**
Whole navigation is provided with tab key within the left navigation pane which is increasing the efforts of users to completely navigate with tab key within the complete left navigation pane & then to reach to main section.

**Expected:**
For better usability purpose, whole navigation should not be provided with tab key within the left navigation pane as it will increase the efforts of users to completely navigate with tab key & then to reach to main section. navigation should be provided with up/down arrow keys.

**MAG Guideline:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7bF98F4235-B6BD-47D3-8DA4-F28B9B6FBFA8%7d&file=Guideline%20101%20%E2%80%93%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties.docx&action=default&mobileredirect=true

**Attachment:**

[Usable_Homepage_left navigation.zip](https://github.com/microsoft/BotFramework-Composer/files/4235928/Usable_Homepage_left.navigation.zip)

**Comment by corinagum** [comment]
Can we also add the severity label to this issue?

**Comment by corinagum** [comment]
Hi @ashish315, my understanding is that interactable items should always be tabable. Because of this, shouldn't both arrow keys and tab work for navigating the side panel?

**Comment by corinagum** [comment]
@ashish315 ping

**Comment by Padmini171991** [comment]
@corinagum Giving tab focus to left nav controls is increasing effort by navigating the same controls all the whenever the user needs to navigate to right-hand side page controls.  Thus arrow navigation helps to skip left nav controls when the user just wants to interact with right-hand side page controls.
e.g: https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html

**Comment by corinagum** [comment]
Thanks!

**Comment by corinagum** [comment]
To developer: it looks like the `<Header>` and `<NavItem>`s will need to have the appropriate roles added, as well as script to handle key presses. the menubar example above is a great resource.

**Comment by ashish315** [comment]
As checked the issue is resolved now. So, HCL is closing the issue
